src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1311,6 +1311,44 @@ export class ResultsViewPageStore {
         },
     });
 
+
+    readonly remoteNgchmUrl = remoteData<string>({
+        await:()=>[this.studyIds],
+        invoke:async()=>{
+			var	result = '';
+
+			if (this.studyIds.result!.length === 1) {
```

Comment:
whats going on with the formatting in this block?

Reply:
Some tabs snuck in there.  Is that what you mean?  I've now detabbed that section.
Thanks!